rest/route/github.go

```diff
@@ -105,8 +105,10 @@ func (gh *githubHookApi) Parse(ctx context.Context, r *http.Request) error {
 // shouldSkipWebhook returns true if the event is from a GitHub app and the app is available for the owner/repo or,
 // the event is from webhooks and the app is not available for the owner/repo.
 func (gh *githubHookApi) shouldSkipWebhook(ctx context.Context, owner, repo string, fromApp bool) bool {
-	token, _ := gh.settings.CreateInstallationToken(ctx, owner, repo, nil)
-	hasApp := token != ""
+	hasApp, err := gh.settings.HasGitHubApp(ctx, owner, repo)
+	if err != nil {
```

Comment:
I think we can keep the original logic; it doesn't look like we return hasApp = true if there's an error (which seems correct -- that should be an error)